Source/Core/Core/Core.cpp

```diff
@@ -806,6 +801,17 @@ static bool PauseAndLock(bool do_lock, bool unpause_on_unlock)
   return was_unpaused;
 }
 
+static void ResetRumble()
```

Comment:
You have to put this higher in the file, before it's used for the first time.

Reply:
I forgot to add the method to Core.h too. I'll do that.

Reply:
You don't need to (and I don't think you should) because this is a static function.